docs/docs-beta/docs/guides/deployment/docker.md

```diff
@@ -1,3 +1,197 @@
 ---
-title: "Deploying to Docker"
+title: "Deploying with Docker"
+description: A guide to deploying Dagster with Docker.
 ---
+
+This guide provides instructions for deploying Dagster using Docker. It's common to use Docker containers to run the Dagster web server, daemon, and code servers. These containers are typically deployed using AWS ECS or another container-based orchestration system.
+
+## What you'll learn
+
+- The different Docker containers that run as part of a Dagster Docker deployment
+- How to define Docker images for each of these containers
+- How to write a docker-compose file that stands up these containers
+
+<details>
+  <summary>Prerequisites</summary>
+- Familiarity with Docker
+</details>
+
+### Understand the Docker containers that run as part of a Dagster Docker deployment
```

Comment:
This strikes me as more conceptual than guide, but I get why you'd want to include it. I think you could rework the intro a little bit to be more specific about the typical containers:

> This guide provides instructions for deploying Dagster using Docker. **A typical Dagster Docker deployment includes containers for the Dagster web server, daemon, code servers, and individual runs.** These containers are typically deployed using AWS ECS or another container-based orchestration system.

Then this section could be scrapped entirely, as much of the info here is already included in other sections. Ex:

> The Dagster web server and daemon are the two host processes in a Dagster deployment. **They typically each run in their own container, using the same Docker image.** This image contains Dagster packages and configuration, but no user code.

Reply:
👍